backend/src/test/java/de/neuefische/cgnjava222/productgallery/service/FileIntegrationTest.java

```diff
@@ -59,40 +71,65 @@ void uploadFile() throws Exception {
                 )
         ).thenReturn(Map.of("url", "hi", "public_id", "bla"));
 
-        MockMvc mockMvc
-                = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
-        mockMvc.perform(multipart("/api/image/uploadFile/").file(firstFile))
+        mockMvc.perform(multipart("/api/image/uploadFile/").file(firstFile).with(csrf()))
                 .andExpect(status().isCreated());
     }
 
+    // @Test
+    // @WithMockUser(username = "frank", authorities = {"ADMIN", "USER"})
+    // void uploadFileExceptionsFileNameNull() {
+    //     MockMultipartFile nullNamedFile = new MockMultipartFile(
+    //             "file", null,
+    //             MediaType.TEXT_PLAIN_VALUE,
+    //             "Hello, World!".getBytes());
+
+    //     Exception exception = assertThrows(Exception.class, () -> {
+    //         mockMvc.perform(multipart("/api/image/uploadFile/").file(nullNamedFile).with(csrf()));
+    //     });
+
+    //     String expectedMessage = "File Upload der Datei: wurde nicht durchgeführt";
+    //     String actualMessage = exception.getMessage();
+
+    //     assertThat(actualMessage).contains(expectedMessage);
+    // }
+
     @Test
-    void uploadFileExceptionsFileNameNull() {
-        MockMultipartFile nullNamedFile = new MockMultipartFile(
-                "file", null,
+    @WithAnonymousUser
+    void tryToAddImageAsAnonymousUser() throws Exception {
+        MockMultipartFile firstFile = new MockMultipartFile(
+                "file", "sawIcon.png",
                 MediaType.TEXT_PLAIN_VALUE,
                 "Hello, World!".getBytes());
-        MockMvc mockMvc
-                = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
-        Exception exception = assertThrows(Exception.class, () -> {
-            mockMvc.perform(multipart("/api/image/uploadFile/").file(nullNamedFile));
-        });
-
-        String expectedMessage = "File Upload der Datei: wurde nicht durchgeführt";
-        String actualMessage = exception.getMessage();
-
-        assertThat(actualMessage).contains(expectedMessage);
+        Assertions.assertNotNull(firstFile);
```

Comment:
You just initialied firstFile with a new object, it will never be null so there is no need to assert that